stubs/playsound/playsound.pyi

```diff
@@ -0,0 +1,7 @@
+from typing import Any
+
+logger: Any
```

Comment:
We should probably remove this. It seems to be an implementation detail, and not something anyone should ever import.